make-util.js

```diff
@@ -174,12 +174,12 @@ var buildNodeTask = function (taskPath, outDir) {
             fail('The package.json should not contain dev dependencies other than typescript. Move the dev dependencies into a package.json file under the Tests sub-folder. Offending package.json: ' + packageJsonPath);
         }
 
-        run('npm ci');
+        run('npm install');
     }
 
     if (test('-f', rp(path.join('Tests', 'package.json')))) {
         cd(rp('Tests'));
-        run('npm ci');
+        run('npm install');
```

Comment:
(Not blocking for this PR) -- need to figure out npm ci vs npm install.  Sounds like there is a conflict with appcenter tasks using npm install?